parachain/pallets/inbound-queue/src/test.rs

```diff
@@ -164,7 +157,7 @@ impl SendXcm for MockXcmSender {
 		match dest {
 			Some(MultiLocation { interior, .. }) => {
 				if let X1(Parachain(1001)) = interior {
-					return Err(XcmpSendError::NotApplicable)
+					return Err(XcmpSendError::NotApplicable);
```

Comment:
Are you sure your formatting settings are correct?

Both of my formatter's are not making this change:
`cargo +$RUST_NIGHTLY_VERSION fmt`
```"rust-analyzer.rustfmt.overrideCommand": [
        "rustup",
        "run",
        "nightly-2023-05-23",
        "--",
        "rustfmt",
        "--"
    ],
```

Reply:
Apologies, I had trouble with my local setup where the formatter didn't work in the `polkadot-sdk` submodule, so I turned it off temporarily but forgot to turn it back in. Fixed in Fixed in https://github.com/Snowfork/snowbridge/pull/972/commits/cc9b1fc2aa6f50764287be80db7eb4e8666d4fe7..

Reply:
Looks like your formatter is acting weird again, for example in in `pallets/ethereum-beacon-client/src/lib.rs` 😄